packages/gas-fee-controller/src/gas-util.ts

```diff
@@ -33,17 +33,21 @@ export function normalizeGWEIDecimalNumbers(n: string | number) {
  * Fetch gas estimates from the given URL.
  *
  * @param url - The gas estimate URL.
+ * @param infuraAuthToken - The infura auth token to use for the request.
  * @param clientId - The client ID used to identify to the API who is asking for estimates.
  * @returns The gas estimates.
  */
 export async function fetchGasEstimates(
   url: string,
+  infuraAuthToken: string,
   clientId?: string,
 ): Promise<GasFeeEstimates> {
-  const estimates = await handleFetch(
-    url,
-    clientId ? { headers: makeClientIdHeader(clientId) } : undefined,
-  );
+  const estimates = await handleFetch(url, {
+    headers: {
+      Authorization: `Basic ${infuraAuthToken}`,
+      ...(clientId && makeClientIdHeader(clientId)),
```

Comment:
Note that this does something slightly different than the previous logic, since `clientId` can be a string and `''` is falsy.

Reply:
Good call, I wasn't paying attention since it's been used the other function below. 
Also this doesn't sound right to be added as a empty string, hence I am changing this function slightly for both.

Reply:
Fair enough!